.coveragerc

```diff
@@ -22,6 +22,7 @@ exclude_lines =
     pass
     import
     raise
+    def reraise
```

Comment:
This file is for statements that occur multiple times, which is not the case here. And `reraise` is covered anyway, so we can remove that line.

```suggestion
```